src/main/java/mate/academy/Main.java

```diff
@@ -6,16 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
-        User user;
+        User user = null;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
-            return;
+            logger.error("Can't login user with login: {}", "bob", e);
```

Comment:
Let's make our log messages informative. Include the stack trace in the log message for better debugging. Good example: logger.error("Can't login user with login: {}", "bob", e);

Reply:
don't remove `return`, it changes execution flow of the program